scripts/before_install.sh

```diff
@@ -7,11 +7,15 @@ mkdir -p /home/datamade/la-metro-councilmatic
 
 if [ "$DEPLOYMENT_GROUP_NAME" == "staging" ]
 then
+    # Preserve uploaded PDFs
+    mv /home/datamade/lametro-staging/lametro/static/pdf/agenda-*.pdf /tmp
```

Comment:
Presumably the change to `after_install` is these getting moved back? (Juuuuuuust checking! 😇)

Reply:
Yep!